engine/classes/Elgg/Application.php

```diff
@@ -207,6 +221,13 @@ public function bootCore() {
 			_elgg_input_init();
 			_elgg_nav_init();
 
+			$file = Paths::project() . 'elgg-services.php';
```

Comment:
Wouldn't we want to have public core services in the Elgg root instead of project root?

Reply:
This is an experimental no-database boot. I could just drop this for now.